test/ReadmeTest.jl

```diff
@@ -110,13 +110,17 @@ using ApproxFun, SpecialFunctions, LinearAlgebra, Test
 
 
         QR = qr([Dirichlet(d);Laplacian()+100I])
-                @time ApproxFun.resizedata!(QR,:,4000)
-                @time u = \(QR,
-                                [ones(∂(d));0.];tolerance=1E-7)
-
-
-        @test u(0.1,1.) ≈ 1.0
-        @test ≈(u(0.1,0.2),-0.02768276827514463;atol=1E-8)
+        # this is broken on v1.6 (on BandedArrays < v0.16.16), so we skip on errors
+        try
```

Comment:
Why not just do:
```suggestion
if VERSION ≥ v"1.7"
            @time ApproxFun.resizedata!(QR,:,4000)
             @time u = \(QR, [ones(∂(d)); 0.]; tolerance=1E-7)
             @test u(0.1,1.) ≈ 1.0
             @test ≈(u(0.1,0.2),-0.02768276827514463;atol=1E-8)
end
```

Reply:
I think you are overthinking things... my proposal is a more definitive statement: "this feature is only supported in Julia 1.7 and up". If it happens to work in 1.6 is not so important.

Reply:
Updated now